docs/en-us/dev/user_doc/guide/task/datax.md

```diff
@@ -1,17 +1,64 @@
+-    **Task priority**: When the number of worker threads is insufficient, they are executed in order from high to low, and when the priority is the same, they are executed according to the first-in first-out principle.
+-    **Worker grouping**: Tasks are assigned to the machines of the worker group to execute. If Default is selected, a worker machine will be randomly selected for execution.
+-    **Environment Name**: Configure the environment name in which to run the script.
+-    **Number of failed retry attempts**: The number of times the task failed to be resubmitted.
+-    **Failed retry interval**: The time, in cents, interval for resubmitting the task after a failed task.
+-    **Delayed execution time**: The time, in cents, that a task is delayed in execution.
+-    **Timeout alarm**: Check the timeout alarm and timeout failure. When the task exceeds the "timeout period", an alarm email will be sent and the task execution will fail.
+-    **Custom template**: Custom the content of the DataX node's json profile when the default data source provided does not meet the required requirements.
+-    **json**: json configuration file for DataX synchronization.
+-    **Custom parameters**: SQL task type, and stored procedure is a custom parameter order to set values for the method. The custom parameter type and data type are the same as the stored procedure task type. The difference is that the SQL task type custom parameter will replace the \${variable} in the SQL statement.
+-    **Data source**: Select the data source from which the data will be extracted.
+-    **sql statement**: the sql statement used to extract data from the target database, the sql query column name is automatically parsed when the node is executed, and mapped to the target table synchronization column name. When the source table and target table column names are inconsistent, they can be converted by column alias.
+
+- **Target library**: Select the target library for data synchronization.
+- **Pre-sql**: Pre-sql is executed before the sql statement (executed by the target library).
+- **Post-sql**: Post-sql is executed after the sql statement (executed by the target library).
+- **Stream limit (number of bytes)**: Limits the number of bytes in the query.
+- **Limit flow (number of records)**: Limit the number of records for a query.
+- **Running memory**: the minimum and maximum memory required can be configured to suit the actual production environment.
+- **Predecessor task**: Selecting a predecessor task for the current task will set the selected predecessor task as upstream of the current task.
+
+## Task Example
+
+This example demonstrates importing data from Hive into MySQL.
+
+### Configuring the DataX environment in DolphinScheduler
+
+If you are using the DataX task type in a production environment, it is necessary to configure the required environment first. The configuration file is as follows: `/dolphinscheduler/conf/env/dolphinscheduler_env.sh`.
```

Comment:
👍 Good catch, I think we forget this in previous docs

Reply:
> 👍 Good catch, I think we forget this in previous docs

OK, I will update it by another pr.

Reply:
No, in this time, we should only change `dev` docs, previous content just let it go.